{"abstract":[{"text":"The error returned from the attachment upload attempt.","type":"text"}],"variants":[{"traits":[{"interfaceLanguage":"occ"}],"paths":["\/documentation\/smiclientcore\/smierrorattachmentuploadresult\/error"]}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorAttachmentUploadResult"]]},"identifier":{"url":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorAttachmentUploadResult\/error","interfaceLanguage":"occ"},"schemaVersion":{"minor":3,"major":0,"patch":0},"sections":[],"metadata":{"role":"symbol","fragments":[{"kind":"identifier","text":"error"}],"externalID":"c:objc(pl)SMIErrorAttachmentUploadResult(py)error","modules":[{"name":"SMIClientCore"}],"navigatorTitle":[{"text":"error","kind":"identifier"}],"symbolKind":"property","title":"error","roleHeading":"Instance Property"},"primaryContentSections":[{"declarations":[{"languages":["occ"],"tokens":[{"kind":"keyword","text":"@property"},{"kind":"text","text":" ("},{"kind":"keyword","text":"nonatomic"},{"kind":"text","text":", "},{"kind":"keyword","text":"strong"},{"kind":"text","text":", "},{"kind":"keyword","text":"readonly"},{"kind":"text","text":") "},{"kind":"typeIdentifier","preciseIdentifier":"c:objc(cs)NSError","text":"NSError"},{"kind":"text","text":" * "},{"kind":"identifier","text":"error"},{"kind":"text","text":";"}],"platforms":["iOS"]}],"kind":"declarations"}],"references":{"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore":{"title":"SMIClientCore","type":"topic","abstract":[],"url":"\/documentation\/smiclientcore","role":"collection","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore","kind":"symbol"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIErrorAttachmentUploadResult":{"abstract":[{"text":"Represents an error result from an attachment upload attempt.","type":"text"},{"text":" ","type":"text"},{"text":"This protocol extends SMIAttachmentUploadResult to provide additional error-specific information.","type":"text"}],"fragments":[{"text":"SMIErrorAttachmentUploadResult","kind":"identifier"}],"title":"SMIErrorAttachmentUploadResult","type":"topic","kind":"symbol","identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorAttachmentUploadResult","role":"symbol","url":"\/documentation\/smiclientcore\/smierrorattachmentuploadresult"},"doc://com.salesforce.SMIClientCore/documentation/SMIClientCore/SMIErrorAttachmentUploadResult/error":{"identifier":"doc:\/\/com.salesforce.SMIClientCore\/documentation\/SMIClientCore\/SMIErrorAttachmentUploadResult\/error","abstract":[{"text":"The error returned from the attachment upload attempt.","type":"text"}],"fragments":[{"text":"error","kind":"identifier"}],"role":"symbol","kind":"symbol","url":"\/documentation\/smiclientcore\/smierrorattachmentuploadresult\/error","title":"error","type":"topic"}}}